src/pages/documentation/mobile-core/identity/api-reference.md

```diff
@@ -168,26 +168,18 @@ Flutter
 
 ## registerExtension
 
+<InlineAlert variant="warning" slots="text"/>
+
+This API is only available in Android and is deprecated starting in version 2.0.0 of the Identity extension. Use [`MobileCore.registerExtensions()`](../../api-reference.md#registerextensions) instead.
+
 The `registerExtension()` API registers the Identity extension with the Mobile Core extension. This API allows the extension to send and receive events to and from the Mobile SDK.
 
-<TabsBlock orientation="horizontal" slots="heading, content" repeat="2"/>
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="1"/>
 
 Android
 
 <Tabs query="platform=android&api=register-extension"/>
 
-iOS
-
-<Tabs query="platform=ios&api=register-extension"/>
-
-<!--- React Native
-
-<Tabs query="platform=react-native&api=register-extension"/>
-
-Flutter
-
-<Tabs query="platform=flutter&api=register-extension"/> --->
```

Comment:
Here too for re-adding the comment blocks.

Reply:
We have removed `registerExtension` APIs in the latest React Native and Flutter wrappers. It should be good to remove those lines now.

Reply:
Oooh got it. Yup, let's remove it then!